CppCoreGuidelines.md

```diff
@@ -21643,7 +21643,7 @@ To avoid confusing macros with names that obey scope and type rules.
 
 ##### Note
 
-This rule applies to non-macro symbolic constants:
+This rule does not apply to non-macro symbolic constants (see also [Enum.5: Don't use `ALL_CAPS` for enumerators](#Renum-caps)):
```

Comment:
I feel it's awkward either way, rule goes "Use X only for A" and now as a note "This does not apply to B"

Maybe more like "In particular, this avoids confusing macros with non-macro symbolic constants (see also...)"